src/auth/auth.module.ts

```diff
@@ -0,0 +1,18 @@
+import { Module } from '@nestjs/common';
+import { AuthService } from './auth.service';
+import { AuthController } from './auth.controller';
+import { AccessTokenStrategy, RefreshTokenStrategy } from './strategy';
+import { UsersService } from 'src/users/users.service';
```

Comment:
Please try to avoid this 'src/' import, it can cause some build problems

Reply:
to avoid it and to shorten the import path size you can config path mapper in tsconfig.json file
to map folder "./src" as "@src/"
in some cases where path may be '../../../../../../src/..." so this mapping allows us to transform this path to "@src/" independently of a place to use

Reply:
```
    "paths": {
      "@src/*": ["src/*"]                      // Set up @src alias for your src folder
    },
```